cmd/minikube/cmd/start_flags.go

```diff
@@ -208,6 +211,9 @@ func initMinikubeFlags() {
 	startCmd.Flags().String(staticIP, "", "Set a static IP for the minikube cluster, the IP must be: private, IPv4, and the last octet must be between 2 and 254, for example 192.168.200.200 (Docker and Podman drivers only)")
 	startCmd.Flags().StringP(gpus, "g", "", "Allow pods to use your NVIDIA GPUs. Options include: [all,nvidia] (Docker driver with Docker container-runtime only)")
 	startCmd.Flags().Duration(autoPauseInterval, time.Minute*1, "Duration of inactivity before the minikube VM is paused (default 1m0s)")
+	startCmd.Flags().String(qemuProgram, "", "The program used to start QEMU. If left empty automatically decides.")
```

Comment:
does it automatically "decide" or does it use the default value?

Reply:
It chooses a value depending on the machine architecture